Fabric.Authorization.AccessControl/src/app/modules/access-control/member-list/member-list.component.ts

```diff
@@ -123,25 +117,39 @@ export class MemberListComponent implements OnInit {
       });
   }
 
+  confirmRemoveRolesFromMember(member: IAuthMemberSearchResult) {
```

Comment:
If this function is unused, can we remove it?

Reply:
@michaelvidal24 done.